deployments/data-hub/release-data-hub--prod.yaml

```diff
@@ -99,6 +99,8 @@ spec:
       - name: CIVICRM_EMAIL_DATA_CONFIG_FILE_PATH
         value: /dag_config_files/civicrm-email-report-data-pipeline.config.yaml
       # scheduler interval
+      - name: ELIFE_ARTICLE_XML_PIPELINE_SCHEDULE_INTERVAL
+        value: "@hourly" # It will be changed after initial load.
```

Comment:
Is this initial load done in staging? If so, it should be okay to just copy the table to prod before the first run?

Reply:
Yes it is done, ok I will copy the table then and change the sch interval

Reply:
Done.